salt/files/minion.d/f_defaults.conf

```diff
@@ -43,6 +43,12 @@ master:
 # choose a random master
 {{ get_config('random_master', 'False') }}
 
+# Set the master type
+{{ get_config('master_type', 'str') }}
```

Comment:
Oh, the docs fail to mention that there's a third (default) option... Just connect to the master that's listed like normal (which is what 'str' means... lazy devs).

This is good, but that option needs documentation. Would you mind opening an issue in the main salt repository that asks for the default/'str' option to be documented?